server/app/routes/api/v1/assessments/rubrics.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+var router = require('express').Router();
+var _ = require('lodash');
+
+const sequelizeHandlers = require( 'sequelize-handlers' );
+const bluebird = require( 'bluebird' );
+const utils = require('../../../utils');
+const {ensureAuthenticated, ensureIsAdmin, Credentials, respondWith404, _err, db} = utils;
+const models = db().models;
+const {
```

Comment:
Why is this one done differently than all of the previous models in the previous routes?